pkg/discovery/network/openshift4.go

```diff
@@ -90,13 +95,18 @@ func parseOS4Network(cr *unstructured.Unstructured) (*ClusterNetwork, error) {
 		result.ServiceCIDRs = append(result.ServiceCIDRs, serviceNetwork.(string))
 	}
 
-	result.NetworkPlugin, found, err = unstructured.NestedString(cr.Object, "spec", "networkType")
-
+	ocpNetworkType, found, err := unstructured.NestedString(cr.Object, "spec", "networkType")
 	if err != nil {
 		return nil, errors.Wrap(err, "error retrieving spec.networkType field")
 	} else if !found {
 		return nil, fmt.Errorf("field .spec.networkType expected, but not found in Network resource: %v", cr.Object)
 	}
+	// calico plugin named 'Calico' in networkType
```

Comment:
I think we probably shouldn't link comments to values of variables like this, to me it seems more likely to cause maintenance issues than to be helpful.

Reply:
:+1: 
Comment has been deleted.